docs/guide/api-javascript.md

```diff
@@ -138,6 +139,10 @@ interface ViteDevServer {
    * Stop the server.
    */
   close(): Promise<void>
+  /**
+   * Bind shortcuts
+   */
+  bindShortcuts(options?: BindShortcutsOptions): void
```

Comment:
This leaves `customShortcuts` available. I'm not against it. Adding new one will not be breaking because custom ones will match first

Reply:
Thanks for guiding me on this PR.

I'm not sure what needs to be modified. If you want to optimize, you can help me modify directly

Reply:
Yeah I can guide you if we want to change this. Waiting for @bluwy opinion
Thanks for the other updates!

Reply:
Hmm yeah I guess that would be nice too. Astro is also hoping to leverage this new API to enable shortcuts for the server, so extending with custom shortcuts is very likely too. So I'm fine with this for now.